app/javascript/packs/vue/books_list.vue

```diff
@@ -1,31 +1,31 @@
 <template>
-  <div id="book">
-    <ul v-for="book in books">
-      <h5> {{book.title}} </h5>
-      <p> {{book.description}} </p>
-    </ul>
+  <div>
+    <div id="books-list">
+      <book-info v-for="book in books" :key="book.id"
```

Comment:
You should use self closing tag here since you don't show any text inside `book-info`